REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.2-
 URIParser
-SHA
\ No newline at end of file
+SHA
+Compat
```

Comment:
You need to add the lower bound compat version
Compat 0.3.6

It would also be good to add the min versions of URIParser and SHA to get rid of dep warnings in these libraries
URIParser 0.0.5
SHA 0.0.4